VoodooGPIO/VoodooGPIO.hpp

```diff
@@ -80,8 +80,8 @@ class VoodooGPIO : public IOService {
 
     IOWorkLoop* getWorkLoop();
 
- public:
-    virtual IOReturn setInterruptTypeForPin(int pin, int type);
```

Comment:
Making this a pure virtual will  require any classes inheriting from this to define it. I am not sure if that's a good idea.

Reply:
All GPIO classes must implement this